tests/core/index_management/test_index_management.py

```diff
@@ -276,3 +279,85 @@ def test_get_marqo_version_v20_successful(self):
             )
 
             self.assertEqual(self.index_management.get_marqo_version(), '2.0')
+
+    def test_createAndDeleteIndexCannotBeConcurrent(self):
+        """Test to ensure create_index and delete_index is not concurrent"""
```

Comment:
Description doesn't make  sense. What does it mean that they're not concurrent?

Reply:
Update the doc string